src/Http/Controllers/FilepondController.php

```diff
@@ -29,15 +29,15 @@ public function __construct(Filepond $filepond)
      */
     public function upload(Request $request)
     {
-        $file = $request->file('file');
-
-        $filePath = tempnam(config('filepond.temporary_files_path'), "laravel-filepond");
-        $filePathParts = pathinfo($filePath);
-
-        if(!$file->move($filePathParts['dirname'], $filePathParts['basename'])) {
-            return Response::make('Could not save file', 500);
+        foreach ($request->file('file') as $file) {
+            $filePath = tempnam(config('filepond.temporary_files_path'), "laravel-filepond");
+            $filePathParts = pathinfo($filePath);
+
+            if (!$file->move($filePathParts['dirname'], $filePathParts['basename'])) {
+                return Response::make('Could not save file', 500);
+            }
+            return Response::make($this->filepond->getServerIdFromPath($filePath), 200);
```

Comment:
This can't work, can it? The return inside of the foreach loop would end the function execution after moving the first image.